src/components/InternalOverlay/__tests__/internalOverlay.spec.js

```diff
@@ -85,4 +85,59 @@ describe('<InternalOverlay />', () => {
         );
         expect(mockContentMetaResolver).toHaveBeenCalledTimes(1);
     });
+
+    it('should call positionResolver when it is passed', () => {
+        const positionResolverMock = jest.fn();
+        mount(
+            <InternalOverlay
+                render={() => <div id="test-id" />}
+                triggerElementRef={{}}
+                positionResolver={positionResolverMock}
+                isVisible
+            />,
+        );
+        expect(positionResolverMock).toHaveBeenCalledTimes(1);
```

Comment:
I think we should remove this test since the below one already test that the position resolver was called